### ISSUE TITLE
Bug: custom logo upload does not work in .189.

### ISSUE THREAD

**Issue by matthijsvp** [MISP/MISP#9671]
### Actual behavior

Uploading a logo file to be used on the welcome page / site headers /etcetera works, but when trying to use the files the following error pops up:
![image](https://github.com/MISP/MISP/assets/14289956/962b90dd-4df1-4495-81bf-ab1dc07a9ba7)

There was reference to this in https://github.com/MISP/MISP/pull/9575, but I'm unsure if this merged fix is actually in the release already. In .189 the functionality still is broken.

### Expected behavior

Logo upload through webinterface and configuring uploaded images for use should work as intended.

### Steps to reproduce

1. Upload a logo through the UI
2. Try to use that logo on – for example – the welcome screen.

### Version

2.4.189

### Operating System

Ubuntu

### Operating System version

22.04

### PHP version

7.4.33

### Browser

N/A

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by iglocska** [comment]
I saw that @JakubOnderka pushed a fix for this already on develop, so it will be in the upcoming release, sorry for the inconvenience.

**Comment by matthijsvp** [comment]
Gotcha, I saw that that PR was merged but wasn't sure in what version it should've been. Is there a way I could've known?

**Comment by Kagee** [comment]
Whatever release was the first one after the PR was merged. A release is basicly just promoting everything in dev to 2.4.

**Comment by matthijsvp** [comment]
Yeah but it was merged the 20th of February and there were some releases after that, so that's what made me question myself and resubmit.

**Comment by Kagee** [comment]
Yes, this should be in .189. I can't see any relevant commits in current develop - https://github.com/MISP/MISP/compare/2.4...develop - @iglocska, are you sure this is not a new bug? 🤔

**Comment by iglocska** [comment]
You are right, it was indeed a commit from february, not sure why I came across it again the other day. Will check it out what went wrong here
